isis/src/base/objs/OriginalXmlLabel/OriginalXmlLabel.cpp

```diff
@@ -20,8 +20,7 @@ namespace Isis {
   /**
    * Constructors a default OriginalXmlLabel with an empty label.
    */
-  OriginalXmlLabel::OriginalXmlLabel() : Isis::Blob("IsisCube", "OriginalXmlLabel") {
-    p_blobPvl += Isis::PvlKeyword("ByteOrder", "NULL");
```

Comment:
We may still want this in the default constructor but I can't give a good reason why 😆

Reply:
I kinda tried, but I wasn't sure how to make that happen... p_blobPvl is a member of Blob, and there's no associated Blob at the point where this is called.  Do you have any ideas for how to make that work?

Reply:
I think we want to move this to the toBlob method. Updating the blob label that we return the blob form of the xmllabel